src/test/scala/com/ambiata/poacher/hdfs/HdfsSpec.scala

```diff
@@ -51,4 +61,54 @@ class HdfsSpec extends Specification { def is = s2"""
     v ==== 1
   }
 
+  def moveFileCheck = prop((source: HdfsTemporary, dest: HdfsTemporary, contents: String) => (for {
```

Comment:
> You should only need one HdfsTemporary as a prop, multiple calls to HdfsTemporary#path should return unique paths each time

It does, but you'll get the exact same number of subdirs. I think it's probably better to leave it the way it is...